01-lesson_notebooks/01a-Some_Context.ipynb

```diff
@@ -39,16 +39,7 @@
     "\n",
     "Programming languages, on the other hand, are a higher-level set of instructions that are designed to be more human-readable and easier to write and understand. They use keywords, variables, and other elements that are familiar to programmers, and are not directly executed by the CPU. Instead, a program written in a high-level language must be translated into machine code before it can be executed by the computer. This translation process is typically done by a compiler or interpreter.\n",
     "\n",
-    "So, programming languages are used by developers to write programs in a way that is more readable and maintainable for humans, but then the code written in programming languages needs to be translated into machine language so that computer can understand it and execute it.\n",
-    "\n",
-    "### Compiled vs. Interpreted languages\n",
```

Comment:
can we skip talking about interpreted languages?

Reply:
Sure, I can see the point that it would be a bit intimidating. I have the section `03-additional_material`. We could cut `Compiled vs Interpreted Languages` section from this workbook and give it its own workbook in `03-additional_material`. Then we an simply link to it for those who want to explore the difference between a compiled and interpreted language.